Errorprone warning: JdkObsolete for Vector

The warning:

```
/.../google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBlobTableModel.java:65: warning: [JdkObsolete] Vector performs synchronization that is usually unnecessary; prefer ArrayList.
        buildDataVector(this.blobs, directoryPrefix), new Vector<>(TABLE_COL_HEADER));
                                                      ^
    (see http://errorprone.info/bugpattern/JdkObsolete)
/.../google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBlobTableModel.java:73: warning: [JdkObsolete] Vector performs synchronization that is usually unnecessary; prefer ArrayList.
              Vector<String> rowData = new Vector<>();
                                       ^
    (see http://errorprone.info/bugpattern/JdkObsolete)
```

I'll send a PR to fix this.

--- a/google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBlobTableModel.java
+++ b/google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBlobTableModel.java
@@ -54,6 +54,7 @@ final class GcsBlobTableModel extends DefaultTableModel {
     }
   }
 
+  @SuppressWarnings("JdkObsolete")
   void setDataVector(List<Blob> blobs, String directoryPrefix) {
     this.blobs =
         blobs
@@ -65,6 +66,7 @@ final class GcsBlobTableModel extends DefaultTableModel {
         buildDataVector(this.blobs, directoryPrefix), new Vector<>(TABLE_COL_HEADER));
   }
 
+  @SuppressWarnings("JdkObsolete")
   private static Vector<Vector<String>> buildDataVector(List<Blob> blobs, String directoryPrefix) {
     return blobs
         .stream()

